src/test/SumsTests.scala

```diff
@@ -4,6 +4,76 @@ import magnolia1.*
 import magnolia1.examples.*
 import scala.annotation.StaticAnnotation
 
+object SumsTests:
```

Comment:
I moved the whole `SumsTests` to reenable `derive all subtypes in complex hierarchy` test. Yes, it's [a compiler bug](https://github.com/lampepfl/dotty/issues/17034) but such a seemingly irrelevant change makes it go away.

Reply:
* compiler feature ;)

Sure thing, looks good